xmtp/src/storage/encrypted_store/mod.rs

```diff
@@ -360,6 +360,40 @@ impl EncryptedMessageStore {
             .execute(conn)?;
         Ok(())
     }
+
+    pub fn fetch_and_lock_outbound_payloads(
+        &self,
+        payload_state: OutboundPayloadState,
+        lock_duration_ns: i64,
+    ) -> Result<Vec<StoredOutboundPayload>, StorageError> {
+        let conn = &mut self.conn()?;
+        use self::schema::outbound_payloads::dsl::*;
```

Comment:
DUST: From a readability level I would suggest using a scoped import rather than a wildcard, I've been finding it easy to run into name conflicts between fields in the model and params being passed into the function

https://github.com/xmtp/libxmtp/blob/9a4b8ccaddcea7cb76d3edf0177af55ad079fe1c/xmtp/src/storage/encrypted_store/mod.rs#L183

Reply:
Oh ya. I've definitely run into that as well